doc/build/kconfig/setting.rst

```diff
@@ -198,6 +198,55 @@ configuration that gets modified when making changes in the :ref:`interactive
 configuration interfaces <menuconfig>`.
 
 
+Tracking Kconfig symbols
+************************
+
+It is possible to create Kconfig symbols which takes the default value of
+another Kconfig symbol.
+
+This is valuable when you want symbol specific for an application or subsystem
+but not rely directly on the common symbol, for example because you would like
+to decouple the settings so that they may be independently configured or to
+ensure you always have a locally name setting, even if the external setting name
```

Comment:
```suggestion
This is valuable when you a want symbol specific for an application or subsystem
but not relying directly on a common symbol, for example because you would like
to decouple the settings so that they may be independently configured or to
ensure you always have a locally name setting, even if the external setting name
```

Reply:
@aescolar this is wrong, the original is fine

Reply:
@erwango those changes are ok

Reply:
suggestion from @erwango applied.

Much clearer, thanks 👍